README.md

```diff
@@ -106,6 +106,11 @@ _Arguments_:
           </tr>
         </thead>
         <tr>
+          <td>enableSoftPwm (optional)</td>
+          <td>boolean</td>
+          <td>Use a software-based approach to PWM on GPIO pins that do not support hardware PWM. The [`raspi-soft-pwm` library](https://github.com/tralves/raspi-soft-pwm) is used to enable this. The default value is <code>false</code>.</td>
```

Comment:
Let's add another sentence at the end: "<em>Note:</em> the timing of software PWM may not be as accurate as hardware PWM.